tests/queries/test_q.py

```diff
@@ -3,6 +3,30 @@
 
 
 class QTests(SimpleTestCase):
+    def test_and_falsey(self):
```

Comment:
While making a few cosmetic edits, I noticed that this test is passing without the fix. If it's just adding additional test coverage, I'll merge it as a separate commit.